proto/provenance/marker/v1/tx.proto

```diff
@@ -38,8 +40,27 @@ service Msg {
   rpc Transfer(MsgTransferRequest) returns (MsgTransferResponse);
   // Allows Denom Metadata (see bank module) to be set for the Marker's Denom
   rpc SetDenomMetadata(MsgSetDenomMetadataRequest) returns (MsgSetDenomMetadataResponse);
+  // GrantAllowance grants fee allowance to the grantee on the granter's
+  // account with the provided expiration time.
+  rpc GrantAllowance(MsgGrantAllowance) returns (MsgGrantAllowanceResponse);
 }
 
+// MsgGrantAllowance adds permission for Grantee to spend up to Allowance
+// of fees from the account of Granter.
+message MsgGrantAllowance {
```

Comment:
Should we rename this to MsgGrantAllowanceRequest?

Reply:
Yes, this isn't the allowance itself, it is a request to create an allowance.